docs/code/code-indexer.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[code-indexer]]
+== Indexes
+
+Code indexes all imported repositories in order to keep them up to date. Indexes are a list of files that Code tracks differences between overtime. There are two types of indexes:
+
+- **Full:** Indexes all files from the upstream repository’s HEAD branch. This occurs when importing a repository for the first time or manually reindexing.
+- **Incremental:** Indexes files that contain differences between the clone and the upstream HEAD branch. Occurs daily by default, but can be customized.
+
+[float]
+==== Perform a full reindex
+
+To perform a full reindex of your repository, go to the Code app, find the repository you want to update and click **Reindex**. The amount of time it takes to perform a full reindex depends on your cloud configuration and the amount of data being indexed. All existing data is cleaned and replaced with the latest code from the upstream HEAD branch. This <<code-semantic-nav, data>> includes:
+
+- **Document:** Each file in the repository
```

Comment:
nit: `Each file in the repository` -> `The content of reach file in the repository`.